### ISSUE TITLE
Bug 54061172: [React Native Gallery -> Clipboard]: Screen reader is not announcing the success message after activating the ‘Paste text from the Clipboard' button.

### ISSUE THREAD

**Issue by Yajur-Grover** [microsoft/react-native-gallery#494]
Build Details:
OS Version: Dev (OS Build 27695.1000)
App: React Native Gallery 
App Version: 1.0.18.0
React Native Windows Version: 0.75.0
AT: Screen Reader
 
Pre-Requisite:
Turn on Narrator (Ctrl + Win + Enter).
 
Repro-Steps:
Launch the React Native Gallery app.
Navigate to ‘All samples’ button present in left navigation pane and invoke it.
Navigate to ‘Clipboard’ button present under “System” group and invoke it.
Navigate to ‘Copy text to the Clipboard’ button and listen to narrator announcement.
Actual Result:
Upon activating 'Paste text from Clipboard' button, narrator fails to announce the success information to the user. When user invoke the ‘Paste text from Clipboard' button, narrator stays on mute.
 
Note: Same issue is observed upon activating ‘Copy text to the Clipboard' button.
 
Expected Result:
Narrator should announce the success information to the user after activating the 'Paste text from Clipboard' button and it should announce like "Text pasted from clipboard". So that, user can understand clearly.
 
User Impact:
Users with visual impairments who rely on screen reader will not be able to understand if the narrator is not announcing the status message after activating the button.
 
Link to ADO bug: https://dev.azure.com/microsoft/OS/_workitems/edit/54061172
